### PR TITLE
mm-131 - ability to close the jitsi window in embedded mode

### PR DESCRIPTION
**Pull request by maisnamrajusingh**
#### Summary
Pressing the escape button will close the active meeting window. I have tested this on the web app. I think it may not work on the desktop app. Maybe we can create a separate PR for that if needed. 
#### Ticket Link
https://github.com/mattermost/mattermost-plugin-jitsi/issues/131

### CODE REVIEW

File: webapp/src/components/conference/conference.tsx [hunk 1]
```diff
@@ -326,7 +334,9 @@ export default class Conference extends React.PureComponent<Props, State> {
         return (
             <React.Fragment>
                 <div
+                    onChange={this.escFunction}
```
Comment (larkox) [inline webapp/src/components/conference/conference.tsx:337]: Is this needed? When would it be triggered?

File: webapp/src/components/conference/conference.tsx [hunk 2]
```diff
@@ -326,7 +334,9 @@ export default class Conference extends React.PureComponent<Props, State> {
...
                     id='jitsiMeet'
+
```
Comment (larkox) [inline webapp/src/components/conference/conference.tsx:339]: Unneeded change?

File: webapp/src/components/conference/conference.tsx [hunk 3]
```diff
@@ -401,5 +411,6 @@ function getStyle(height: number, width: number, position: 'top' | 'bottom'): {[
             opacity: 0.85,
             zIndex: buttonsZIndex
         }
+
```
Comment (larkox) [inline webapp/src/components/conference/conference.tsx:414]: Unneeded change?

File: webapp/src/components/conference/conference.tsx [hunk 4]
```diff
@@ -128,6 +135,7 @@ export default class Conference extends React.PureComponent<Props, State> {
     }
 
     componentDidMount() {
+        document.addEventListener('keydown', this.escFunction, false);
```
Comment (larkox) [inline webapp/src/components/conference/conference.tsx:138]: Shouldn't we also remove the listener on componentDidUnmount?

**Review by larkox — CHANGES_REQUESTED** [review]
Thanks for the changes! Some minor considerations.

File: webapp/src/components/conference/conference.tsx (hunk 4 above)
Comment (sanjaydemansol) [reply webapp/src/components/conference/conference.tsx:138]: done

File: webapp/src/components/conference/conference.tsx (hunk 3 above)
Comment (sanjaydemansol) [reply webapp/src/components/conference/conference.tsx]: removed

File: webapp/src/components/conference/conference.tsx (hunk 2 above)
Comment (sanjaydemansol) [reply webapp/src/components/conference/conference.tsx]: removed

File: webapp/src/components/conference/conference.tsx (hunk 1 above)
Comment (sanjaydemansol) [reply webapp/src/components/conference/conference.tsx]: removed..

**Review by larkox — APPROVED** [review]
LGTM

File: webapp/src/components/conference/conference.tsx [hunk 5]
```diff
@@ -53,6 +54,12 @@ export default class Conference extends React.PureComponent<Props, State> {
         return Math.max(document.documentElement.clientHeight || 0, window?.innerHeight || 0) - (BORDER_SIZE * 2);
     }
 
+  escFunction = (event: any) => {
+      if (event.keyCode === 27) {
```
Comment (catalintomai) [inline webapp/src/components/conference/conference.tsx:58]: add a comment that '27' == escape key code?

**Review by catalintomai — APPROVED** [review]
small comment, but otherwise good to go.

File: webapp/src/components/conference/conference.tsx (hunk 5 above)
Comment (maisnamraju) [reply webapp/src/components/conference/conference.tsx:58]: will update it. Thanks

**Comment by larkox** [comment]
@DHaussermann heads up that this has been merged expecting testing during release testing.

**Pull request merged**